### PR TITLE
T2 - CEPH-11314 - Backup and restore data using existing NFS backup t…

### PR DESCRIPTION
**Pull request by hkadam134**
…ools

Logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-NHJOTN/
Duration: 5min 14sec.

Signed-off-by: Hemanth <hyelloji@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/nfs/data_backup_restore_bw_nfs_and_local_mounts.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        run_ios(
+            clients[0], nfs_mounting_dir, file_name="dd_file_nfs2", bs="100M", count=20
+        )
+
+        log.info("Backup the data from NFS mounts to local dir")
+        nfspath1 = f"{nfs_mounting_dir}{clients[0].node.hostname}dd_file_nfs1"
+        nfspath2 = f"{nfs_mounting_dir}{clients[0].node.hostname}dd_file_nfs2"
+        commands = [
+            f"rsync {nfspath1} {local_dir}",
+            f"scp {nfspath2} {local_dir}",
```
Comment (yogesh-mane) [inline tests/cephfs/nfs/data_backup_restore_bw_nfs_and_local_mounts.py:108]: Could we also add remote copying to other clients from nfs mount using rsync & scp & vice-versa

File: tests/cephfs/nfs/data_backup_restore_bw_nfs_and_local_mounts.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+            check_ec=False,
+        )
+
+
+def run_ios(client, mounting_dir, file_name, bs, count):
+    client.exec_command(
+        sudo=True,
+        cmd=f"dd if=/dev/zero of={mounting_dir}{client.node.hostname}{file_name} bs={bs} "
+        f"count={count}",
+    )
```
Comment (yogesh-mane) [inline tests/cephfs/nfs/data_backup_restore_bw_nfs_and_local_mounts.py:164]: I think we already have this function in move_data_bw_nfs_and_cephfs_mounts.py

File: tests/cephfs/nfs/data_backup_restore_bw_nfs_and_local_mounts.py (hunk 1 above)
Comment (hkadam134) [reply tests/cephfs/nfs/data_backup_restore_bw_nfs_and_local_mounts.py:108]: I think there is another TC which has this remote copy.

**Comment by psathyan** [comment]
Could we close on this PR? It is pending for quite some time.

**Comment by hkadam134** [comment]
Fixed and re-ran the script
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-2RN022

Will re-submit the PR in some time

**Pull request closed without merging**